### PR TITLE
Added logging middleware for the technical requirements part

### PR DESCRIPTION
**Pull request by IgnasJo**

### CODE REVIEW

File: psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    {
+        _next = next;
+        _options = options;
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        if (_options.CurrentValue.LoggingEnabled) {
+
+            // Get user information
```
Comment (Domantas1337) [inline psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs:24]: What is this comment

File: psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                    Timestamp: {DateTime.Now},
+                    User: {userName},
+                    Permissions: {userPermissions},
+                    HTTP Method: {httpMethodName},
+                    ControllerClass: {invokedClassName},
+                    Method: {invokedMethodName}
+                ------------------------------------------------------------
+                """;
+
+                // Log to file
```
Comment (Domantas1337) [inline psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs:52]: Seems obvious from the method name

File: psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs (hunk 1 above)
Comment (IgnasJo) [reply psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs:24]: That this section of code is about getting user information from HttpContext

**Review by Domantas1337 — APPROVED** [review]

File: psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs (hunk 1 above)
Comment (IgnasJo) [reply psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs:24]: That this section of code is about getting user information from HttpContext

Comment (Domantas1337) [reply psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs:24]: ahhh, I see

**Pull request merged**